tools/block-generator/upload_metrics.py

```diff
@@ -0,0 +1,53 @@
+from datadog import initialize
```

Comment:
nit: can combine below import
```suggestion
from datadog import api, initialize
```